agasc/supplement/magnitudes/mag_estimate.py

```diff
@@ -722,18 +722,19 @@ def calc_obs_stats(telem):
 
     kalman = (telem['AOACASEQ'] == 'KALM') & (telem['AOPCADMD'] == 'NPNT')
     track = (telem['AOACIIR'] == 'OK') & (telem['AOACFCT'] == 'TRAK')
-    dr3 = (telem['dr'] < 3)
-    dr5 = (telem['dr'] < 5)
+    dr3 = (telem['dy'] < 3) | (telem['dz'] < 3)
```

Comment:
I would have never guessed that, but if it helps I will change it. We would be changing only `dr5` to use a square metric.

Still... strictly speaking these flags do not have to correspond to whatever the OBC was thinking the residual was. I believe that adding the square cut exactly is not going to make it agree with the OBC expectation more because the centroid positions might be different.

In any case, if you *really* want the square cut, I will change it.